src/bindings/c/include/openvino/c/ov_infer_request.h

```diff
@@ -70,37 +71,149 @@ OPENVINO_C_API(ov_status_e)
 ov_infer_request_set_tensor(ov_infer_request_t* infer_request, const char* tensor_name, const ov_tensor_t* tensor);
 
 /**
- * @brief Sets an input tensor to infer on.
+ * @brief Sets an input/output tensor to infer on by the port of input/output tensor.
```

Comment:
Port it is not a part of tensor, it is a model part. I think this sentence can be changed to:
```suggestion
 * @brief Sets an input/output tensor to infer for the port.
```